src/content/docs/synthetics/synthetic-monitoring/getting-started/types-synthetic-monitors.mdx

```diff
@@ -70,6 +70,26 @@ These are the four types of synthetic monitors:
       </td>
     </tr>
 
+    <tr>
+      <td>
+        Step monitor
+
+        API name: `STEP_MONITOR`
+      </td>
+
+      <td>
+        Step Monitors are advanced monitors which require no code to set up. 
+
+        The monitor can be configured to:
+        - Navigate to a URL 
+        - Type text
+        - Click an element
+        - Assert text
```

Comment:
s/be "assert" or "insert" for this bullet and next?

Reply:
It should be "assert text" as that's the UI text